ara/views/playbook.py

```diff
@@ -47,15 +49,15 @@ def playbook_results(playbook):
                     .filter(models.Playbook.id == playbook.id)
                     .order_by(models.TaskResult.time_start))
 
-    if request.args.get('host'):
-        hosts = [str(host) for host in request.args.get('host').split(',')]
+    if host is not None:
+        hosts = [str(h) for h in host.split(',')]
```

Comment:
:cry: How do I request /playbook/result/<status>?  That is, what if I want to see all results for all hosts, but limited by status?  What if I want to see all results for `localhost` with status `ok` or `changed`, rather than just one or the other?  I don't think we should be crippling the web ui just to make static generation more convenient.